c/extras/py_ftrl.cc

```diff
@@ -217,11 +189,11 @@ Train an FTRL model on a dataset.
 Parameters
 ----------
 X: Frame
-    Datatable frame of shape (nrows, ncols) to be trained on. 
+    Datatable frame of shape (nrows, ncols) to be trained on.
```

Comment:
So should we say `datatable frame` or just `frame` in documentation and error messages?

Reply:
I see, but there are other packages that provide different types of frames, like for example Pandas, right? Or we assume if a user imports `datatable`, she or he knows what is meant by frame here?

Reply:
If the user is reading `datatable` documentation, we can assume that he/she can guess that the subject is datatable Frame :) The only exception may be when we're talking about interoperability with other libraries, for example `pandas`. Then it could make sense to explicitly say "pandas DataFrame" or "datatable Frame".

Reply:
Ok, let's hope for the best :) I will then clean all the places like this.